stacks/linkerd2/deploy.sh

```diff
@@ -23,12 +23,20 @@ wget -q $URL -O "$BINARY" && chmod +x "$BINARY"
 kubectl config set-context --current --namespace=linkerd
 
 # deploy linkerd
+$BINARY install --crds --ignore-cluster | kubectl apply -f -
 $BINARY install --ignore-cluster | kubectl apply -f -
 
 # ensure services are running
+kubectl get deployments -o custom-columns=NAME:.metadata.name
+
 kubectl get deployments -o custom-columns=NAME:.metadata.name | tail -n +2 | while read -r line
```

Comment:
Is there a reason you're running this command twice? Same with the install command on line 26 and 27.

Reply:
no, I used for debugging and forgot to remove